front_end/html/navbar_course.html

```diff
@@ -7,7 +7,7 @@
                     {% if len(courses) > 0 %}
                         <div id="moreDropdown" class="navbar-dropdown is-size-4">
                             {% for course in courses %}
-                                <a class="navbar-item is-active" href="/course/{{ course[0] }}">
+                                <a class="navbar-item" href="/course/{{ course[0] }}">
```

Comment:
I'm not sure if this change was intentional or not. If it was, please let me know what it does. If not, please put is-active back after navbar-item.

Reply:
It was intentional; the is-active wasn't in the assignment and problem navbar pages and was making the the text stay permanently a blue-green color. Without the is-active, it only turns blue-green when it is hovered over, which I think is what we want.

Reply:
Excellent. Thanks for catching it!